File: CSIS 113A C++ Level 1/Lab10_11/Lab10_11_2Rectangle/Rectangle.cpp
#include <iostream>
#include <string>
#include "Rectangle.h"

using namespace std;

void Rectangle::set_values(int l, int w) 
{
	length = l;
	width = w;
}
int Rectangle::area() const
{
	return length * width;
}
void Rectangle::set_name(string nm) 
{
	name = nm;
}

File: CSIS 113B Java Level 1/java/Java/src/main/java/GP/Rectangle.java
/*
John Barbonio, 3408, GP #8A dependency
0518876
9/29/2019
CSIS 113B
 */
package GP;

/**
 *
 * @author itzdu
 */
public class Rectangle
{
    private int length;
    private int width;

    public void setRectangle(int length, int width) 
    {
        this.length = length;
        this.width = width;
    }
    public int area()   //a = b * h
    {
        return this.length * this.width;
    }
}

